README.md

```diff
@@ -23,9 +23,15 @@ https://napari.org/plugins/index.html
 
 ## Installation
 
-You can install `napari-epitools` via [pip]:
+`napari-epitools` is not yet available to install from PyPI or napari-hub.
 
-    pip install napari-epitools
+To install, please clone the repository and do a local install using `pip`:
+
+```
+git clone git@github.com:epitools/epitools.git
```

Comment:
If we're going to tell the users how to clone it we should probably change this to the `https` version

Reply:
well spotted! I've changed it to use `https`